include/openspace/scene/translation.h

```diff
@@ -46,7 +46,16 @@ class Translation : public properties::PropertyOwner {
 
     glm::dvec3 position(double time);
 
+    // Registers a callback that gets called when a significant change has been made that
+    // invalidates potentially stored points, for example in trails
+    void onParameterChange(std::function<void()> callback);
```

Comment:
Yeah, I agree.. and maybe Michael and Sebastian's event system is the way to go... at the moment my thought was that those objects are only owned by a single objects anyway, so there is not going to be an issue about it.. but yeah.. something to keep in mind!